x-pack/plugins/apm/typings/get_transaction.ts

```diff
@@ -0,0 +1,14 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { Transaction } from 'x-pack/plugins/apm/typings/es_schemas/Transaction';
+
+export type TransactionAPIResponse = Transaction | undefined;
+
+export interface TransactionWithErrorCountAPIResponse {
+  transaction: TransactionAPIResponse;
+  errorCount: number;
+}
```

Comment:
I think these should be located by the API they are describing. This is the approach used for all other APIs, eg: https://github.com/elastic/kibana/blob/43c938b5fa67b322fa2db8004f7ef0908e7fe53d/x-pack/plugins/apm/server/lib/errors/distribution/get_distribution.ts.

Accessing them from the client should be fine, since typescript runs at compile time. None of this ever ends up in the code shipped to the client.